docker/scripts/entrypoint.sh

```diff
@@ -148,10 +167,27 @@ if [ "${DEV_MODE:-false}" != "true" ]; then
   validate_and_download "ZKV_CONF_CHAIN" "ZKV_SPEC_FILE_URL"
 fi
 
-ZKV_SECRET_PHRASE_PATH=${ZKV_SECRET_PHRASE_PATH:-"/data/config/secret_phrase.dat"}
-ZKV_NODE_KEY_FILE=${ZKV_NODE_KEY_FILE:-"/data/config/node_key.dat"}
-echo "  ZKV_SECRET_PHRASE_PATH=${ZKV_SECRET_PHRASE_PATH}"
-echo -e "  ZKV_NODE_KEY_FILE=${ZKV_NODE_KEY_FILE}\n"
+# Creating node key file if node key is provided
+if [ -n "${ZKV_NODE_KEY}" ]; then
+  ZKV_NODE_KEY_FILE="/tmp/node_key.dat"
+
+  printf "%s" "${ZKV_NODE_KEY}" > "${ZKV_NODE_KEY_FILE}"
+  chmod 0400 "${ZKV_NODE_KEY_FILE}" && chown "${RUN_USER}" "${ZKV_NODE_KEY_FILE}"
+  echo -e "  ZKV_NODE_KEY_FILE=${ZKV_NODE_KEY_FILE}\n"
+
+  unset ZKV_NODE_KEY
+fi
+
+# Creating secret phrase file if secret phrase is provided
+if [ -n "${ZKV_SECRET_PHRASE}" ]; then
+  ZKV_SECRET_PHRASE_FILE="/tmp/secret_phrase.dat"
```

Comment:
See above.

Reply:
Discussed in person. Switching to `/dev/shm/` location for both secrets without option to change the location to enforce RAMDISK